test/libsolidity/SemanticTest.cpp

```diff
@@ -40,12 +40,21 @@ using namespace boost::unit_test;
 namespace fs = boost::filesystem;
 
 
-SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVersion, vector<boost::filesystem::path> const& _vmPaths, bool enforceViaYul):
+SemanticTest::SemanticTest(
+	string const& _filename,
+	langutil::EVMVersion _evmVersion,
+	vector<boost::filesystem::path> const& _vmPaths,
+	bool _enforceViaYul,
```

Comment:
We could think about moving those into a setting struct.

Reply:
Can we do it in separate PR, this one is already huge? I can do it as a follow up to this one.

Reply:
Sure!